deploy/kubernetes/manifests/catalogue-db-dep.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
```

Comment:
In general it's useful to put `---` at the top of yamels, so they can be 'cated together.